src/main/java/org/jabref/logic/shared/DBMSConnection.java

```diff
@@ -10,10 +10,10 @@
 import org.jabref.logic.shared.exception.InvalidDBMSConnectionPropertiesException;
 import org.jabref.model.database.shared.DBMSType;
 import org.jabref.model.database.shared.DatabaseConnection;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
 public class DBMSConnection implements DatabaseConnection {
```

Comment:
Hey, @dimmonn. It looks that there is a problem with the Checkstyle test. You should insert a blank line before org.slf4j.Logger and remove one of the two empty lines after org.slf4j.LoggerFactory:

> import org.jabref.model.database.shared.DatabaseConnection;
> 
> import org.slf4j.Logger;
> import org.slf4j.LoggerFactory;
> 
> public class DBMSConnection implements DatabaseConnection {

Thanks for your contribution!

Reply:
Got you, many thanks will fix now